pycytominer/tests/test_cyto_utils/test_cells.py

```diff
@@ -416,6 +434,48 @@ def test_merge_single_cells():
         traditional_norm_df.loc[:, new_compartment_cols].abs().describe(),
     )
 
+    # use non-default float_datatype
+    sc_merged_df = AP.merge_single_cells(float_datatype=np.float32)
```

Comment:
I'm a bit confused why this test is necessary? What are we testing here?

Reply:
Thank you for the check here - I added this in to make certain the `float_datatype` work could cascade correctly from `merge_single_cells` in addition to `load_compartment`. With the changes you suggested for the class attribute I think this may no longer be relevant, so I'll remove it.

Reply:
got it!

It's probably sufficient to assert `self.float_datatype` once you make the class attribute change